packages/mermaid/src/diagrams/architecture/architectureRenderer.ts

```diff
@@ -379,6 +379,15 @@ function layoutArchitecture(
           },
         },
       ],
+      layout: {
+        name: 'grid',
+        boundingBox: {
+          x1: 0,
+          x2: 100,
+          y1: 0,
+          y2: 100,
+        },
+      },
```

Comment:
The rendering test fails when using D3 directly on a JSDOM if there's no bounding box here.
I'm not sure this fix is OK though.